onnxUI.py

```diff
@@ -11,9 +11,18 @@
     OnnxRuntimeModel,
     OnnxStableDiffusionPipeline,
     OnnxStableDiffusionImg2ImgPipeline,
+    OnnxStableDiffusionInpaintPipeline,
+    OnnxStableDiffusionInpaintPipelineLegacy,
     DDIMScheduler,
     PNDMScheduler,
     LMSDiscreteScheduler,
+    EulerDiscreteScheduler,
+    EulerAncestralDiscreteScheduler,
+    DPMSolverMultistepScheduler,
+    DPMSolverSinglestepScheduler,
+    DEISMultistepScheduler,

```

Comment:
importing DEISMultistepScheduler will fail if ran with diffusers older than 0.12.0. Hence why later on I checked the version then imported the missing schedulers.

Reply:
Good point. I quickly redid the original 0.8.0 check to be the 0.12.0, moved those (>0.8.0 <0.12.0) schedulers up and intended to then make the import conditional for new schedulers. Yet then after testing which ones I could get to output a viable result, I forgot to put them in the right place. Probably shouldn't have finished it at 2AM ...